apps/test/unit/code-studio/pd/workshop_dashboard/components/workshop_managementTest.js

```diff
@@ -117,20 +117,20 @@ describe('WorkshopManagement', () => {
     it('uses organizer results for organizers', () => {
```

Comment:
Should we delete these tests? I'm not sure what their purposes are now.

Reply:
Good question.  I'm neutral.  For what it's worth, the `getSurveyUrlForProps` function still returns valid values in some tests, and so these tests do exercise the removals in this PR.